lib/msteams_hermes/components/adaptive_card.rb

```diff
@@ -14,7 +14,12 @@ module Components
     class AdaptiveCard < Base
       attr_reader :body, :actions, :entities, :width
 
-      def initialize(body:, actions: nil, entities: [], full_width: false)
+      WIDTH = {
+        default: "default",
+        full: "full"
+      }.freeze
+
+      def initialize(body:, actions: nil, entities: [], width: WIDTH[:default])
```

Comment:
The idea is to also use a guard to protect against someone using a not accepted constant/string.

```
raise "You can only use constants defined in  <MSTAdaptiveCardWidth>" unless MSTAdaptiveCardWidth.all.include? width
```